internal/builder/packages.go

```diff
@@ -150,6 +173,18 @@ func BuildPackage(options BuildOptions) (string, error) {
 		return "", errors.Wrap(err, "clearing package contents failed")
 	}
 
+	m, err := packages.ReadPackageManifestFromPackageRoot(options.PackageRoot)
+	if err != nil {
+		return "", errors.Wrapf(err, "reading package manifest failed (path: %s)", options.PackageRoot)
+	}
+	if license := m.Source.License; license != "" {
+		logger.Debugf("Write license text for %q", license)
+		err := licenses.WriteTextToFile(license, filepath.Join(destinationDir, "LICENSE.txt"))
```

Comment:
Is there an option to use your own license? Will it break the building logic here?

Reply:
Not at the moment, do we want to support it?

If we want, we could relax the check of unknown licenses, and include whatever the developer places in the `source.license` field and the `LICENSE.txt` file.

Reply:
(it will break the current logic because it assumes that `source.license` if set, is set to one of the values enumerated in the spec)